assets/css/sensei-course-theme/base.scss

```diff
@@ -1,27 +1,22 @@
+@import 'colors';
+
 $breakpoint: 783px;
 
 body {
-	// Try to pick up global styles, customizer or theme colors.
-	--sensei-primary-color: var(--sensei-primary-color-global, var(--sensei-course-theme-primary-color, var(--wp--preset--color--primary, #307771)));
-	--sensei-background-color: var(--sensei-background-color-global, var(--sensei-course-theme-background-color, var(--wp--preset--color--background, #ffffff)));
-	--sensei-primary-contrast-color: var(--sensei-background-color, #FFFFFF);
-	--sensei-text-color: var(--sensei-text-color-global, var(--sensei-course-theme-foreground-color, var(--wp--preset--color--text, var(--wp--preset--color--foreground, #1E1E1E))));
 	--border-color: rgba(125, 125, 125, 0.3);
-
 	background-color: var(--sensei-background-color);
 	color: var(--sensei-text-color);
 }
 
 .sensei-course-theme__frame {
 	--wp--style--block-gap: 0px;
 	box-sizing: border-box;
+	color: $sensei-module-lesson;
+	font-size: 13px;
```

Comment:
Should we use `0.8125rem` instead?

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6783/commits/e9ca39ea7b403486b9751a601d1b706210a53d79